src/scenes/AcceleratorRouter/AcceleratorCohortNewView.tsx

```diff
@@ -47,10 +47,10 @@ export default function AcceleratorCohortNewView(): JSX.Element {
       // set snackbar with status
       snackbar.toastSuccess(strings.formatString(strings.COHORT_ADDED, cohort.name) as string);
 
-      // navigate to accelerator overview
-      goToAcceleratorOverview();
+      // navigate to cohorts list
+      goToCohortsList();
```

Comment:
Should we be going to the newly created cohort view? Just to to stick with current pattern.

Reply:
Ah, yes, good catch

Reply:
Gonna make this change in #2219 to avoid conflicts